onnxruntime/core/framework/session_state.cc

```diff
@@ -903,7 +903,19 @@ common::Status SessionState::GetInputNodeInfo(const std::string& input_name,
                                               InlinedVector<NodeInfo>& node_info_vec) const {
   auto entry = input_names_to_nodeinfo_mapping_.find(input_name);
   if (entry == input_names_to_nodeinfo_mapping_.cend()) {
-    return Status(ONNXRUNTIME, FAIL, "Failed to find input name in the mapping: " + input_name);
```

Comment:
What is this change? Why is this change necessary?

Reply:
Hi kainan, the purpose of this code is to use string matching when the map key(Type:String) cannot be found, because some of the model's test data provided by onnxruntime itself **on the previous branch** cannot be found due to **inconsistent naming**. However, if you think this hack method is not formal, I will restore the source code.